pod/perlre.pod

```diff
@@ -1350,24 +1350,60 @@ X</p> X<p modifier>
 
 =head2 Quoting metacharacters
 
-Backslashed metacharacters in Perl are alphanumeric, such as C<\b>,
-C<\w>, C<\n>.  Unlike some other regular expression languages, there
-are no backslashed symbols that aren't alphanumeric.  So anything
-that looks like C<\\>, C<\(>, C<\)>, C<\[>, C<\]>, C<\{>, or C<\}> is
-always
-interpreted as a literal character, not a metacharacter.  This was
-once used in a common idiom to disable or quote the special meanings
-of regular expression metacharacters in a string that you want to
-use for a pattern. Simply quote all non-"word" characters:
+(Also known as "escaping".)
+
+To cause a metacharacter to match its literal self, you precede it with
+a backslash.  Unlike some other regular expression languages, any
+sequence consisting of a backslash followed by a non-alphanumeric
+matches that non-alphanumeric, literally.  So things like C<\\>, C<\(>,
+C<\)>, C<\[>, C<\]>, C<\{>, or C<\}> are always interpreted as the
+literal character that follows the backslash.
+
+(That's not true when an alphanumeric character is preceded by a
+backslash.  There are a few such "escape sequences", like C<\w>, which have
+special matching behaviors in Perl.  All such are currently limited to
+ASCII-range alphanumerics.)
+
+But a non-alphanumeric will always match literally when preceded by a
+backslash.  Hence simply adding backslashes before all non-"word"
+characters can be used to disable the special meanings of regular
+expression metacharacters in a string that you want to use for a
+pattern. 
 
     $pattern =~ s/(\W)/\\$1/g;
 
-(If C<use locale> is set, then this depends on the current locale.)
-Today it is more common to use the C<L<quotemeta()|perlfunc/quotemeta>>
-function or the C<\Q> metaquoting escape sequence to disable all
-metacharacters' special meanings like this:
+then
 
-    /$unquoted\Q$quoted\E$unquoted/
+    $string =~ s/$pattern/foo/;
+
+(If C<use locale> is in effect, the current locale can affect the
+results.)
+
+This template used to be a common paradigm, but these days it is more
+usual to use the C<L<quotemeta()|perlfunc/quotemeta>>
+function or especially the C<\Q> metaquoting escape sequence to disable all
+metacharacters' special meanings.
+
+In fact, C<quotemeta> effectively does the same thing as the old
+paradigm, but without any locale dependence.  That is, 
+
+    quotemeta $pattern;
+
+does either of these:
```

Comment:
I'm not quite sure this is correct, it's not quite how the quotemeta docs describe it, but it sounds complex. In particular it says that ASCII characters are always subject to the ASCII style escaping regardless of unicode_strings or the utf8 flag. Maybe better to say it does similar things, give the basics, and link to the quotemeta docs for details.

Reply:
You're right, I thought I knew what was going on, but didn't.  I have since RTFM, and the code, and I'm much happier with the newest iteration.

I also added a commit which changes a heading as I originally did (and which I think reads better) and has a workaround in case someone did link to the old wording

Reply:
LGTM